zips/zip-0317.rst

```diff
@@ -95,14 +103,21 @@ Requirements
   two inputs and two outputs for each shielded pool used by the transaction).
 * Users should be able to spend a small number of UTXOs or notes with value
   below the marginal fee per input.
```

Comment:
This is a requirement we failed to mention but it applies to both the existing mechanism and ZSAs. I'll file another PR for it since it is independent of ZSAs.
```suggestion
  below the marginal fee per input.
* The conventional fee should not leak private information used in
  constructing the transaction; that is, it should be computable from
  only the public data of the transaction.
```